pre-processing/primod/driver_coupling/ribamod.py

```diff
@@ -106,12 +106,22 @@ def write_exchanges(
                     save_flows=True,
                 )
                 destination = "river"
+                if isinstance(self, RibaModPassiveDriverCoupling):
+                    raise TypeError(
+                        f"Expected Drainage (no 'active' packages in passive coupling), received: {type(package).__name__}"
```

Comment:
"Expected Drainage packages for passive coupling,  received: {type(package).__name__}" ?